### ISSUE TITLE
Issue about loading data

### ISSUE THREAD

**Issue by EayonWongJunyi** [liruiw/HPT#9]
Hello,
Thanks for your work.
I am trying to finetune the policy on calvin environment. After I implement the `convert_dataset` function for calvin dataset and finish generating data, it takes hundreds of times longer than the metaworld dataset to load the data during training (the `data_time` in `train` function) while the step time is similar.
I think I do generate the right data with the same fomat as your metaworld example. So I wonder if you have met the same problem or I might have missed something?

**Comment by liruiw** [comment]
This is a bit strange. To confirm, convert_dataset is just generating the datasets, but it's the data loading part that is slow in your case. Are you using disks for loading or memory? https://github.com/liruiw/HPT/blob/main/experiments/configs/config.yaml#L34

**Comment by EayonWongJunyi** [comment]
> This is a bit strange. To confirm, convert_dataset is just generating the datasets, but it's the data loading part that is slow in your case. Are you using disks for loading or memory? https://github.com/liruiw/HPT/blob/main/experiments/configs/config.yaml#L34

I have tried both using disk and using memory, and it took the same time to load data. 
I also have tried a tiny subset of the origin dataset with 100 trajs, and got the same data time.
Besides, I found that after initializing the data loaded from the calvin dataset, data key can't be printed correctly and got `data keys: KeysView(<zarr.hierarchy.Group '/data' read-only>)`. Will that be the reason? I can't understand why that happened while the only thing I have done is writing a `convert_dataset` function to yield the `data_dict`.
